### DISCUSSION TITLE
Extra FEA material- Project 4 Page not Loading

### DISCUSSION THREAD

**Discussion by owenlotito** [cooperrc/computational-mechanics#236] [Q&A]
I get this message when I try to open the extra FEA material on Project 4 for part 2.D
<img width="436" alt="fea material error" src="https://github.com/cooperrc/computational-mechanics/assets/154915768/51352510-7846-468d-a6ad-96c24cd64250">

**Comment by cooperrc** [comment]
Try this link, 

https://cooperrc.github.io/computational-mechanics/projects/extra-FEA_material.html
